test/test_jit.py

```diff
@@ -5670,6 +5670,18 @@ def fn(x, y, z):
         ast = torch.jit.frontend.get_jit_def(fn, fn.__name__)
         self.assertExpected(str(ast))
 
+    def test_python_frontend_source_range(self):
+        def fn():
+            raise Exception("hello")
+        ast = torch.jit.frontend.get_jit_def(fn, fn.__name__)
+        src_range_str = str(ast.range())
+        start_quote = src_range_str.find('"')
+        end_quote = src_range_str.find('"', start_quote+1)
+        # ignore the file path string as it might be different on different machines
```

Comment:
What was the error again? was it just a file path difference? Maybe we can look at the error and see if there is a better way of addressing this difference.

Reply:
I believe it is just a file path difference. However, there is also an equality comparison on the line number, which is very undesirable because any change before this test function in `test_jit.py` will very likely break the test. I will figure out a way to do safer testing.